Monika After Story/game/script-greetings.rpy

```diff
@@ -6,37 +6,36 @@
 default persistent._mas_you_chr = False
 
 init python:
-    greetings_list=[]
-    label_list=renpy.get_all_labels()
-    for label in label_list:
-        if label.startswith('greeting_') and not renpy.seen_label(label):
-            greetings_list.append(label)
-
-    #If the greeting's list is empty, remake it without removing seen.
-    if greetings_list == []:
-        for label in label_list:
-            if label.startswith('greeting_'):
-                greetings_list.append(label)
-
+    addEvent(Event(persistent.greeting_database,eventlabel="greeting_sweetheart", random=True),eventdb=evhand.greeting_database)
 
 label greeting_sweetheart:
     m 1k "Hello again, sweetheart!"
     m 1l "It's kind of embarrassing to say out loud, isn't it?"
     m 3b "Still, I think it's okay to be embarrassed every now and then."
     return
 
+init python:
+    addEvent(Event(persistent.greeting_database,eventlabel="greeting_honey", random=True),eventdb=evhand.greeting_database)
```

Comment:
Make these all `unlocked=True`. Greetings should also only appear if they are unlocked.